com.unity.sg2/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Input/Textures/SampleTriplanarNode.cs

```diff
@@ -0,0 +1,193 @@
+using System.Collections.Generic;
+using Usage = UnityEditor.ShaderGraph.GraphDelta.GraphType.Usage;
+
+namespace UnityEditor.ShaderGraph.Defs
+{
+    internal class SampleTriplanarNode : IStandardNode
+    {
+        public static string Name = "SampleTriplanar";
+        public static int Version = 1;
+
+        public static NodeDescriptor NodeDescriptor => new(
+            Version,
+            Name,
+            new FunctionDescriptor[] {
+                new(
+                    1,
+                    "SampleTriplanarDefault",
+@"  Node_UV = Position * Tile;
+    Node_Blend = SafePositivePow_float(Normal, min(Blend, floor(log2(Min_float())/log2(1/sqrt(3)))) );
```

Comment:
Looks like the shader will switch between SafePositivePow_float and [SafePositivePow_half](https://github.com/Unity-Technologies/Graphics/blob/216a86eca4214d7f1aa009649a7967e711b05ac9/Packages/com.unity.render-pipelines.core/ShaderLibrary/Common.hlsl#L685) base on the precision, not sure what will be a good way to do here tho🤔

Reply:
I don't know how to fix it either.

Reply:
we can add a comment here and come back later when we have a way to fix it?

Reply:
Done